src/utils/parsers/currency.js

```diff
@@ -0,0 +1,27 @@
+export const centsToReal = (cents) => {
+  if (typeof cents !== 'number' || !Number.isInteger(cents)) {
+    throw new TypeError('cents must be an Integer number');
```

Comment:
Are you sure about throwing an error without catching that? It can easily cause some issues. If the backend sends an `undefined` amount`, the UI will be broken.

I think it's not a common pattern to throw errors from utils functions, we would have to remember to use `try-catch` every time we use the function.

Since this is a function to format some value, I would suggest adding a default value to return in case the argument is not valid, maybe a 'Not valid amount' or something. Does it make sense?

Here's an example of what the UI will look like in case the argument is not a string:

https://user-images.githubusercontent.com/54550926/167023558-34667b41-8817-43bb-bb67-082f85d895b5.mp4